index.html

```diff
@@ -0,0 +1,228 @@
+
+
+            <div class="hamburger mobile-only">
+                <a href="#mobile-menu"><img src="images/hamburger.png" alt="hamburger" class="hamburgerimg" /></a>
+            </div>
+
+            <!-- mobile menu -->
+
+            <nav class="mobile-menu mobile-only" id="mobile-menu">
+                <!--* close_button -->
+                <img class="close-btn" src="images/close-button.png"alt="close button"/>
+                <ul class="menu-list">
+                    <li class="menu-item-link">
+                    <a
+                        href="./index.html"
+                        class="menu-item"
+                        >Home</a
+                    >
+                    </li>
+                    <li class="menu-item-link">
+                    <a
+                        href="./about.html"
+                        class="menu-item"
+                        >About</a
+                    >
+                    </li>
+                    <li class="menu-item-link">
+                    <a
+                        href="#"
+                        class="menu-item"
+                        >Program</a
+                    >
+                    </li>
+                    <li class="menu-item-link">
+                    <a
+                        href="#"
+                        class="menu-item"
+                        >Join</a
+                    >
+                    </li>
+                    <li class="menu-item-link">
+                    <a
+                        href="#"
+                        class="menu-item"
+                        >Sponsor</a
+                    >
+                    </li>
+                    <li class="menu-item-link">
+                    <a
+                        href="#"
+                        class="menu-item"
+                        >CC Campaign</a
+                    >
+                    </li>
+                </ul>
+            </nav>
+            
+        </header>
+
+        <section class="homepage">
+            <div class="page1-wrap">
+                <div class="page1">
+                    <h1>MICROSOFT GLOBAL </br>
+                        SUMMIT 2023</h1>
+                    <div class="page1-words-wrap">
+                        <p class="page1-words">A joyful celebration believing in the value of openness and sharing, creating a positive change with people from all over 80 countriesis taking place in May, 2023</p>
+                    </div>
+                    <h2>2023.05.15(MON) ~ 16(TUE)</h2>
+                    <p class="page1-location">@ National Theatre Kampala, Uganda</p>
+                </div>
+            </div>
+
+            <div class="page2-wrap">
+                <div class="page2">
+                    <h3 class="page2-title">Main Program</h3>
+                    <hr />
+                    <div class="main-program-container" id="main-program-container">
+                        <div class="main-program-container1"></div>
```

Comment:
- [ ] [OPTIONAL] I am marking this issue as an optional fix because it was not mentioned by the previous reviewer, Please ensure you consider implementing it. 

your design | Template
---- | ----
![image](https://user-images.githubusercontent.com/84156714/212083552-1f050a0f-8b54-4b59-acc2-395009e0a007.png) | ![image](https://user-images.githubusercontent.com/61408860/212030070-750cd10a-446d-4b96-97ad-96cbb2a60c77.png)

Please center-align all the content of each program box.

Reply:
@bobb-Rob Thank you for the time to review my project